blueoil/cmd/main.py

```diff
@@ -172,14 +153,7 @@ def convert(experiment_id, checkpoint, template, image_size, project_name):
     default=True,
 )
 def predict(input, output, experiment_id, checkpoint, save_images):
-    output_dir = os.environ.get('OUTPUT_DIR', 'saved')
-
-    if checkpoint is None:
-        restore_path = None
-    else:
-        restore_path = os.path.join(output_dir, experiment_id, 'checkpoints', checkpoint)
-
-    run_predict(input, output, experiment_id, None, restore_path, save_images)
+    run_predict(input, output, experiment_id, checkpoint, save_images)
```

Comment:
`preduct` function looks pass through argument to `run_predict`. 
The worth of existence of this function is `click.echo` ?
Or You wan to unify interface with `convert` ?

Reply:
Both are correct.
To improve testability, I separated the logic and interface.
The main file is responsible for input from the user and sends message to the user.